### PR TITLE
Add: Github Action

### PR DESCRIPTION
**Pull request by vaibhavsingh97**

### CODE REVIEW

File: nightwatch.conf.js [hunk 1]
```diff
@@ -137,10 +142,11 @@ module.exports = {
       // More info on configuring capabilities can be found on:
       // https://www.browserstack.com/automate/capabilities?tag=selenium-4
       desiredCapabilities: {
+        build: 'nightwatch-website-test',
         'bstack:options': {
           local: 'false',
-          userName: '${BROWSERSTACK_USER}',
-          accessKey: '${BROWSERSTACK_KEY}',
+          userName: process.env.BROWSERSTACK_USER || '${BROWSERSTACK_USER}',
+          accessKey: process.env.BROWSERSTACK_KEY || '${BROWSERSTACK_KEY}',
```
Comment (gravityvi) [inline nightwatch.conf.js]: Nightwatch internally takes env variables into account, this won't be necessary.
For reference: [line](https://github.com/nightwatchjs/nightwatch/blob/5901396607ecec5b5d8cf6be59eff3bd1301dcf6/lib/transport/browserstack.js#L43)

**Review by gravityvi — CHANGES_REQUESTED** [review]

File: nightwatch.conf.js (hunk 1 above)
Comment (vaibhavsingh97) [reply nightwatch.conf.js]: Make sense, removing it 👍

**Review by gravityvi — APPROVED** [review]

**Comment by beatfactor** [comment]
Seems like this is adding a lot more than the github action. Maybe you can leave the other changes out for now?

**Comment by vaibhavsingh97** [comment]
Sure, opening a new branch, as there is a mix-up of commits.

**Comment by vaibhavsingh97** [comment]
New PR: https://github.com/nightwatchjs/nightwatch-website-tests/pull/6

**Pull request closed without merging**